config.yaml

```diff
@@ -86,6 +86,18 @@ options:
       random delay of 25% is added. Changes to this time interval will apply when next interval is
       triggered. Prior to reconciliation, any packages, and services used by the software will be
       updated.
+  repo-policy-compliance-token:
+    type: string
+    description: >-
+      The token to authenticate with the repository-policy-compliance service in order to 
+      generate one-time-tokens. This option requires the repo-policy-compliance-url to be set.
+      If not set, the repository-policy-compliance service will not be used.
+  repo-policy-compliance-url:
+    type: string
+    description: >-
+      The URL to the repository-policy-compliance service. This option requires the 
+      repo-policy-compliance-token to be set. If not set, the repository-policy-compliance service
+      will not be used. This option is currently only effective when using OpenStack Cloud.
```

Comment:
Should these put the charm in blocked state if, for example, only 1 is set or they are set when not using OpenStack?

Reply:
Have updated to go into blocked state when this option is used without OpenStack (code is in `charm_state`)